libfwupd/fwupd-enums-private.h

```diff
@@ -482,5 +482,21 @@ G_BEGIN_DECLS
  * The D-Bus type signature string is 's' i.e. a string.
  **/
 #define FWUPD_RESULT_KEY_VERSION "Version"
+/**
+ * FWUPD_RESULT_KEY_BATTERY_LEVEL:
+ *
+ * Result key to represent the battery level in percent.
+ *
+ * The D-Bus type signature string is 'u' i.e. a unsigned 32 bit integer.
+ **/
+#define FWUPD_RESULT_KEY_BATTERY_LEVEL "BatteryLevel"
+/**
+ * FWUPD_RESULT_KEY_BATTERY_THRESHOLD:
+ *
+ * Result key to represent the battery threshold in percent.
```

Comment:
This should have more description. Threshold of "what"

Reply:
`Result key to represent the battery threshold of the permitted battery level in percent` perhaps?

Reply:
I would say something like
`Result key to represent the minimum battery level required to perform an update.  Expressed from 0-100 %.`